packages/auth/src/Domain/Token/Token.ts

```diff
@@ -12,12 +15,6 @@ export type Token = {
     updated_at: string,
     device_info: string
   },
-  roles: Array<{
-    uuid: Uuid
-    name: string
-  }>,
-  permissions: Array<{
-    uuid: Uuid
-    name: string
-  }>,
+  roles: Array<Role>,
+  features: Array<Feature>,
```

Comment:
Just to double check - is it safe to replace `permissions` by `features` here?

Reply:
yes. This does not take effect until `auth` and `api-gateway` update their version of the `@standardnotes/auth` package